etc/profile-m-z/newsboat.profile

```diff
@@ -8,6 +8,10 @@ include globals.local
 
 noblacklist ${HOME}/.newsboat
 
+# Lynx Support
```

Comment:
Please add what's needed for Lynx to the existing private-bin and private-etc lines in the profile.